tests/test_declare.py

```diff
@@ -42,14 +44,11 @@ class Type(dj.Lookup):
             """
             contents = zip(('Type1', 'Type2', 'Type3'))
 
-        locals()   # this is to overcome the issue described in issue #368
```

Comment:
I'm so happy to see that this is no longer necessary and that the use of `schema` within a function works cleanly.